tests/Handler/CreateAndFetchTest.php

```diff
@@ -96,12 +88,58 @@ public function handleShouldExecuteTheCommandAndReturnAnEmptyResponse(): void
                            ->willReturn('/testing/1');
 
         /** @var ResponseInterface|UnformattedResponse $response */
-        $response = $handler->handle($request);
+        $response = $this->handleRequest($request);
 
         self::assertInstanceOf(UnformattedResponse::class, $response);
         self::assertSame(StatusCodeInterface::STATUS_CREATED, $response->getStatusCode());
         self::assertSame('/testing/1', $response->getHeaderLine('Location'));
         self::assertSame([ExecuteQuery::class => 'query'], $response->getAttributes());
         self::assertSame('result', $response->getUnformattedContent());
     }
+
+    /**
+     * @test
+     *
+     * @uses \Chimera\Routing\HttpRequest
```

Comment:
This misses the coverage information, you should add the `@covers` to the path that we're explicitly using here. Like in https://github.com/chimeraphp/routing/pull/25/files#diff-3eea67346b6f1bf1bbc5f141c192fb6aR61-R65